samples/list_vlan_in_portgroups.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+
+"""
+Written by David Martinez
+Github: https://github.com/dx0xm
+
+Script to list all portgroup vlan on existing or selected dvswitch
+This is for demonstration and reference purposes.
+Testing ok in Single datacenter, multiple dvswitch instances
+Based on getvnicinfo.py and py-vminfo.py
+
+To improve:
+Error handling
+"""
+
+
+from __future__ import print_function
+from pyVim.connect import SmartConnect, Disconnect
+from pyVmomi import vim
+import atexit
+import argparse
+import getpass
+import ssl
+
+
+def get_args():
+    parser = argparse.ArgumentParser(
+        description='Arguments for talking to vCenter')
+
+    parser.add_argument('-s', '--host',
+                        required=True,
+                        action='store',
+                        help='vSpehre service to connect to')
+
+    parser.add_argument('-o', '--port',
+                        type=int,
+                        default=443,
+                        action='store',
+                        help='Port to connect on')
+
+    parser.add_argument('-u', '--user',
+                        required=True,
+                        action='store',
+                        help='User name to use')
+
+    parser.add_argument('-p', '--password',
+                        required=False,
+                        action='store',
+                        help='Password to use')
+
+    parser.add_argument('-d', '--datacenter',
+                        required=True,
+                        help='name of the datacenter')
+
+    parser.add_argument('-dvs', '--dvswitch',
+                        required=False,
+                        help='name of the dvswitch',
+                        default='all')
+
+    parser.add_argument('-c', '--cert_check_skip',
```

Comment:
-S is commonaly used argument for Skip verification. Please can you change to same for consistency. or library argument builder method can be used https://github.com/vmware/pyvmomi-community-samples/blob/master/samples/tools/cli.py

Reply:
Done !